scripts/reporting/create_doi.py

```diff
@@ -0,0 +1,100 @@
+'''
+Programmatically create DOI for a Synapse project
+'''
+import json
+import os
+import requests
+import yaml
+
+''' Constants '''
+OBJECT_ID = 'syn25191261'
+
+def synapse_login():
+    '''
+    Get Oauth2 Access Token for Synapse
+    '''
+    # Get shared log-in credentials
+    config = yaml.load(open(os.path.join(os.path.expanduser("~"),
+                                         ".sibis-general-config.yml")))
+
+    # Necessary request info - URL, payload, and headers
+    login_url = "https://repo-prod.prod.sagebase.org/auth/v1/login2"
+    payload = json.dumps({
+        "username": config.get('synapse').get('user'),
+        "password": config.get('synapse').get('password')
+    })
+    headers = {
+        'Content-Type': 'application/json',
+        'Cookie': 'sessionID=b4dd1b39-d689-4239-b037-10ed0fcd0292'
```

Comment:
Can this really be a constant? I think cookie ID very likely has to be retrieved regularly...